demos/multi_channel/object_detection_demo_yolov3/main.cpp

```diff
@@ -108,12 +109,45 @@ static int EntryIndex(int side, int lcoords, int lclasses, int location, int ent
     return n * side * side * (lcoords + lclasses + 1) + entry * side * side + loc;
 }
 
-struct YoloParams {
-    int num;
-    int classes;
-    int coords;
+class YoloParams {
+    template <typename T>
+    void computeAnchors(const std::vector<float> & initialAnchors, const std::vector<T> & mask) {
+        std::vector<float> maskedAnchors(num * 2);
```

Comment:
This variable is now redundant. Just use `anchors` directly.

Reply:
You didn't update the usages.

Reply:
You don't seem to have changed anything.

Reply:
fixed